applications/nrf_desktop/src/modules/dfu.c

```diff
@@ -352,6 +357,10 @@ static void handle_dfu_start(const uint8_t *data, const size_t size)
 
 	sync_offset = 0;
 
+	if (IS_ENABLED(CONFIG_CAF_POWER_MANAGER_EVENTS)) {
+		power_manager_restrict(MODULE_IDX(MODULE), POWER_MANAGER_LEVEL_SUSPENDED);
```

Comment:
Should be moved after checks, I think right after `DFU started` log

Reply:
Otherwise we may prevent deep sleep forever

Reply:
Good catch! Done.